cmd/admin-handlers-site-replication.go

```diff
@@ -207,8 +207,32 @@ func (a adminAPIHandlers) SRPeerReplicateBucketItem(w http.ResponseWriter, r *ht
 		return
 	}
 
+	types := make([]string, 0, 1)
+	if item.Type != "" {
+		types = append(types, item.Type)
+	} else {
+		types = []string{
+			madmin.SRBucketMetaTypePolicy,
+			madmin.SRBucketMetaTypeQuotaConfig,
+			madmin.SRBucketMetaTypeVersionConfig,
+			madmin.SRBucketMetaTypeTags,
+			madmin.SRBucketMetaTypeObjectLockConfig,
+			madmin.SRBucketMetaTypeSSEConfig,
+		}
+	}
+
+	for _, typ := range types {
+		err := a.doSRPeerReplicateBucketItem(ctx, typ, item)
```

Comment:
we should do multiple types at once as well @vadmeste - we should support that would that require a breaking change?